dev/io.openliberty.microprofile.reactive.streams.operators30.internal/test/src/io/openliberty/microprofile/reactive/streams/test/utils/TestProcessor.java

```diff
@@ -0,0 +1,54 @@
+/*******************************************************************************
+ * Copyright (c) 2023 IBM Corporation and others.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License 2.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-2.0/
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *******************************************************************************/
+package io.openliberty.microprofile.reactive.streams.test.utils;
+
+import org.reactivestreams.Processor;
+import org.reactivestreams.Subscriber;
+import org.reactivestreams.Subscription;
+
+import io.smallrye.mutiny.operators.multi.processors.UnicastProcessor;
+import mutiny.zero.flow.adapters.AdaptersToReactiveStreams;
+
+public class TestProcessor<T> implements Processor<T, T> {
```

Comment:
Not sure what the point of this class is, it just delegates to a mutiny `UnicastProcessor`.

Reply:
It was to avoid changing the actual test too much ... but no big deal